doc/supportedPlatformVersions.md

```diff
@@ -284,7 +288,9 @@
 - 1.16.7
 - 1.17
 - 1.18
+- 1.18.8
 - 1.19
+- 1.19.3
```

Comment:
why we don't need to update the patch version of 1.18 for stretch images?

Reply:
We're only supporting buster and bullseye for the full image, so the stretch flavor isn't needed in this case.